src/App/Fossa/ManualDeps.hs

```diff
@@ -180,10 +217,27 @@ instance FromJSON CustomDependency where
     CustomDependency <$> obj .: "name"
       <*> (unTextLike <$> obj .: "version")
       <*> obj .: "license"
-      <*> obj .:? "description"
-      <*> obj .:? "url"
+      <*> obj .:? "metadata"
```

Comment:
Now that we've moved url away, let's add it to the `forbidMembers` list.

Reply:
great suggestion